src/app/templates/app/submit_new_license.html

```diff
@@ -170,30 +170,29 @@
 {% block script_block %}
 <script type="text/javascript">
 $(document).ready(function () {
-    var is_touch_device = "ontouchstart" in document.documentElement;
-    $('#fullnameInfo').popover({trigger: is_touch_device ? "click" : "hover", title: "License Full Name", content: "\u2022 The full name may omit certain word, such as 'the,' for alphabetical sorting purposes. \
+    $('#fullnameInfo').popover({trigger: "click", title: "License Full Name", content: "\u2022 The full name may omit certain word, such as 'the,' for alphabetical sorting purposes. \
     <br /> \u2022 No commas in full name of license or exception. \
     <br /> \u2022 The word 'version' is not spelled out; 'v' or nothing is used to indicate license version (for space reasons) \
     <br /> \u2022 For version, use lower case v and no period or space between v and the number. \
     <br /> \u2022 No abbreviations are included (in parenthesis) after the full name.", html: "true", viewport: '.panel-body', container: '.panel-body'});
```

Comment:
So much content! Can we save this somewhere like in a file/ javascript variable so it is easy to modify -> for eg if I need to add another point, I have to add `<br />` on every line which is not feasible for non tech people.
Help regarding this would be appreciated.